doctrine/annotations/1.0/config/routes/annotations.yaml

```diff
@@ -0,0 +1,3 @@
+controllers:
+    resource: ../../src/Controller/
```

Comment:
`resource: ../src/Controller/`

Reply:
relative paths are interpreted relative to the location of kernel (probably related to symfony/symfony#24982)

Reply:
But then `resource: Controller/` should be enough